### PR TITLE
integration testing

### PR DESCRIPTION
**Pull request by muradsater**
- fix import typo
- list endpoint has eventual consistency
- Don't fail if delete table fails

### CODE REVIEW

File: .github/workflows/test.yaml [hunk 1]
```diff
@@ -46,6 +46,7 @@ jobs:
         AZURE_CLIENT_ID: ${{ vars.AZURE_CLIENT_ID }}
         AZURE_CLIENT_SECRET: ${{ secrets.AZURE_CLIENT_SECRET }}
         AZURE_TENANT_ID: ${{ vars.AZURE_TENANT_ID }}
+        EXTRAXTOR_RAW_TS_PATH: ${{ vars.EXTRAXTOR_RAW_TS_PATH }}
```
Comment (wbevans) [inline .github/workflows/test.yaml:49]: Misspelled EXTRACTOR

File: tests/integration/integration_steps/fabric_steps.py [hunk 2]
```diff
@@ -39,10 +39,13 @@ def write_timeseries_data_to_fabric(credential: DefaultAzureCredential, data_fra
...
-    DeltaTable(
-        table_uri= table_path,
-        storage_options={"bearer_token": token, "use_fabric_endpoint": "true"}
-    ).delete()
+    try:
+        DeltaTable(
+            table_uri= table_path,
+            storage_options={"bearer_token": token, "use_fabric_endpoint": "true"}
+        ).delete()
+    except Exception as e:
```
Comment (wbevans) [inline tests/integration/integration_steps/fabric_steps.py:47]: Is there a specific error message you can catch instead? Ignoring Exception is very broad and could potentially hide other errors.

**Comment by muradsater** [comment]
There is one test still failing, can you have a look @muralidharm ? Datapoints fetched from CDF is not as expected.

**Comment by muradsater** [comment]
merged into #52

**Pull request closed without merging**